### PR TITLE
fix(developer): support for notany() and context()

### PR DESCRIPTION
**Pull request by mcdurdin**
This is part 1 of 2 of a fix for #917. This adds in version 14.0
targeting and a new function `KNO` which is used in one specific place:
when a context() statement references a notany() statement. This
minimizes any risk in this change because existing supported patterns
should compile identically.

If the compiler finds this pattern, it will enforce 14.0 minimum version
for web targets.

### CODE REVIEW

**Review by jahorton — COMMENTED** [review]
An eyeball test of the changes seems fine.

Since this PR is a package deal with #3817, I'm waiting until I'm ready to approve both of them together.

File: windows/src/developer/TIKE/compile/CompileKeymanWeb.pas
Comment (darcywong00) [inline windows/src/developer/TIKE/compile/CompileKeymanWeb.pas:945]: Should the error inform the user to set `store(&version) '14.0'`?

**Review by jahorton — APPROVED** [review]

File: windows/src/developer/TIKE/compile/CompileKeymanWeb.pas
Comment (mcdurdin) [reply windows/src/developer/TIKE/compile/CompileKeymanWeb.pas:945]: Missed this at the time, but given the error can't really occur I am keeping this lowpri

**Pull request merged**
